tests/integration.spec.coffee

```diff
@@ -104,6 +104,28 @@ describe 'SDK Integration Tests', ->
 	# running the tests in a slow connection
 	@timeout(30 * 60 * 1000)
 
+	describe 'factory function', ->
+
+		validKeys = ['auth', 'models', 'logs', 'settings']
+
+		describe 'given no opts', ->
+
+			it 'should return an object with valid keys', ->
+				mockResin = getSdk()
+				m.chai.expect(mockResin).to.include.keys(validKeys)
```

Comment:
The code for them is so trivial it's not strictly required. If that means exposing the object (to make it reachable from the test code) I'd rather not make this change.